docs/sources/reference/playlist.md

```diff
@@ -11,59 +11,96 @@ weight = 4
 
 # Playlist
 
-The Playlist is a special type of Dashboard that rotates through a list of Dashboards. A Playlist can be a great way to build situational awareness, or just show off your metrics to your team or visitors.
+The Playlist is a special type of Dashboard that rotates through a list of Dashboards. A Playlist can be a great way to build situational awareness, and show off your metrics to your team or visitors.
 
-Since Grafana automatically scales Dashboards to any resolution they're perfect for big screens!
+Grafana automatically scales Dashboards to any resolution, which makes them perfect for big screens.
+
+Access the Playlist feature from Grafana's side menu, in the Dashboards submenu.
 
 ## Create a playlist
 
 {{< docs-imagebox img="/img/docs/v50/playlist.png" max-width="25rem" class="docs-image--right">}}
 
-The Playlist feature can be accessed from Grafana's sidemenu, in the Dashboard submenu.
+To create a playlist: 
```

Comment:
All of my sections are currently formatted like this:
##Do this thing
To do this thing:
(Body)

To confirm, you'd like me to change this to:

##Do this thing
This thing does this and lets you do this. Stuff about this thing.
1. First step
1. Second step

I'll be doing another fun pass to make that change, so that all of these topics can be broken out into their own pages.

Reply:
Correct.